plugins/inventory/proxmox.py

```diff
@@ -31,6 +31,7 @@
         description:
           - URL to Proxmox cluster.
           - If the value is not specified in the inventory configuration, the value of environment variable C(PROXMOX_URL) will be used instead.
+          - Since community.general 4.7.0 you can also use templating to specify the value of the C(url).
```

Comment:
```suggestion
          - Since community.general 4.7.0 you can also use templating to specify the value of the I(url).
```
`I(...)` is for option names, `C(...)` for option values :)